deepchem/feat/material_featurizers/__init__.py

```diff
@@ -5,3 +5,4 @@
 from deepchem.feat.material_featurizers.element_property_fingerprint import ElementPropertyFingerprint
 from deepchem.feat.material_featurizers.sine_coulomb_matrix import SineCoulombMatrix
 from deepchem.feat.material_featurizers.cgcnn_featurizer import CGCNNFeaturizer
+from deepchem.feat.material_featurizers.composition_featurizer import CompositionFeaturizer
```

Comment:
Could you add `CompositionFeaturizer` to the featurization api docs in `docs/source/api_reference/featurizers.rst`?